README.md

```diff
@@ -1,38 +1,38 @@
-[![Skeleton](https://user-images.githubusercontent.com/1509726/199282306-7454adcb-b765-4618-8438-67655a7dee47.png)](https://www.skeleton.dev/)
+[![Shmeleton](https://user-images.githubusercontent.com/1509726/199282306-7454adcb-b765-4618-8438-67655a7dee47.png)](https://www.Shmeleton.dev/)
 
-[![npm version](https://img.shields.io/npm/v/@skeletonlabs/skeleton?logo=npm&color=cb3837)](https://www.npmjs.com/package/@skeletonlabs/skeleton)
+[![npm version](https://img.shields.io/npm/v/@Shmeletonlabs/Shmeleton?logo=npm&color=cb3837)](https://www.npmjs.com/package/@Shmeletonlabs/Shmeleton)
 [![Chat](https://img.shields.io/discord/1003691521280856084?label=chat&logo=discord&color=7289da)](https://discord.gg/EXqV7W8MtY)
-[![license](https://img.shields.io/badge/license-MIT-%23bada55)](https://github.com/skeletonlabs/skeleton/blob/master/LICENSE)
+[![license](https://img.shields.io/badge/license-MIT-%23bada55)](https://github.com/Shmeletonlabs/Shmeleton/blob/master/LICENSE)
 
-# 💀 Skeleton
+# 💀 Shmeleton
 
-Skeleton is an open source UI toolkit built with Svelte + Tailwind that allows you to create reactive web interfaces using [Svelte](https://svelte.dev/) + [Tailwind CSS](https://tailwindcss.com/).
+Shmeleton is an open source UI toolkit built with Svelte + Tailwind that allows you to create reactive web interfaces using [Svelte](https://svelte.dev/) + [Tailwind CSS](https://tailwindcss.com/).
 
 ## 🔗 Useful Links
 
-- [Documentation](https://skeleton.dev/)
-- [Contribution Guide](https://skeleton.dev/docs/contributing)
-<!-- - [Project Roadmap](https://github.com/skeletonlabs/skeleton/wiki/%F0%9F%9B%A3%EF%B8%8F-The-Skeleton-Roadmap) -->
+- [Documentation](https://Shmeleton.dev/)
+- [Contribution Guide](https://Shmeleton.dev/docs/contributing)
+<!-- - [Project Roadmap](https://github.com/Shmeletonlabs/Shmeleton/wiki/%F0%9F%9B%A3%EF%B8%8F-The-Shmeleton-Roadmap) -->
```

Comment:
"Project" a word with the letter J – do we have to? really?

Reply:
Replying in existing conversation

Reply:
I can reply!!!!